fairworkflows/nanopub.py

```diff
@@ -217,16 +217,27 @@ def fetch(uri, format=Format.TRIG):
 
 
     @staticmethod
-    def rdf(assertionrdf, uri=DEFAULT_URI, introduces_concept=None, derived_from=None):
+    def rdf(assertionrdf, uri=DEFAULT_URI, introduces_concept=None, derived_from=None, attributed_to=None, nanopub_author=None):
```

Comment:
```suggestion
    def to_rdf(assertionrdf, uri=DEFAULT_URI, introduces_concept=None, derived_from=None, attributed_to=None, nanopub_author=None):
```

Reply:
Agreed. I've made this change and updated the tests etc that depended on it.